### PR TITLE
Note about Dask Gateway issue

### PR DESCRIPTION
**Pull request by pavithraes**
## What does this implement/fix?

_Put a `x` in the boxes that apply_

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds a feature)
- [ ] Breaking change (fix or feature that would cause existing features not to work as expected)
- [x] Documentation Update
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Other (please describe):

## Testing

- [x] Did you test the pull request locally?
- [x] Did you add new tests?

#### Text styling

- [x] The content is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags (with only one level-one header: `H1` or `#` in markdown).
- [x] All links describe where they link to (for example, check the [Nebari website](https://nebari.dev/)).
- [x] This content adheres to the Nebari style guides.

#### Non-text content

- [x] All content is represented as text (for example, images need alt text, and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing GIFs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If the content were to be read as plain text, it still makes sense, and no information is missing.

## Any other comments?

Include @rsignell-usgs and @viniciusdc as co-authors during sqash merge. :)

### CODE REVIEW

File: docs/docs/troubleshooting.mdx [hunk 1]
```diff
@@ -102,6 +102,17 @@ At this point, feel free to cancel the deployment and rerun the same deployment
...
+### New Dask Gateway clusters lead to `500 Internal Server Error`
+
+This may occur when `dask-gateway-controller` has some internal authentication issues. To resolve this, you can restart `nebari-daskgateway-controller` and `nebari-daskgateway-controller` pods:
+
+- Open `k9s` (Learn more about `k9s` in [Debug Nebari deployment](./how-tos/debug-nebari#debugging-the-kubernetes-cluster-using-k9s))
+- Press the <kbd>/</kbd> key and search for "daskgateway"
+- Use the arrow keys to highlight each pod
+- Delete highlighted pods with <kbd>ctrl</kbd> + <kbd>d</kbd>, and the pods should restart automatically
+
+It's also worth verifying that all API resources are up to date for your Nebari deployment.
```
Comment (pavithraes) [inline docs/docs/troubleshooting.mdx:114]: @viniciusdc I don't fully understand what you meant by this, could you please elaborate a little?

Comment (iameskild) [reply docs/docs/troubleshooting.mdx:114]: I'm not sure about the last sentence but everything else looks good to me.

Comment (pavithraes) [reply docs/docs/troubleshooting.mdx:114]: I'll remove that sentence for now. :)

Comment (pavithraes) [inline docs/docs/troubleshooting.mdx:114]:
```suggestion
```

**Review by iameskild — APPROVED** [review]
LGTM!

**Pull request merged**